### PR TITLE
AC 538: Add Toast Message when going online

### PR DESCRIPTION
**Pull request by Abdelaty**
## Issue I worked on

JIRA Issue: https://issues.openmrs.org/browse/AC-538

## Checklist: I completed these to help reviewers :)

- [ ] My pull request only contains **ONE single commit**
(the number above, next to the 'Commits' tab is 1).

- [ ] I have **added tests** to cover my changes. (If you refactored
existing code that was well tested you do not have to add tests)

- [x] All new and existing **tests passed**.

- [x] My pull request is **based on the latest changes** of the master branch.

### CODE REVIEW

**Comment by Abdelaty** [comment]
@f4ww4z  @Rimjhim28  Can you test it now?

File: openmrs-client/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -341,4 +341,5 @@
     <string name="dialog_button_stay">Stay</string>
     <string name="dialog_button_leave">Leave</string>
     <string name="snackbar_nonvisitting_patients">Vitals can only be captured for patients currently in a visit.</string>
+    <string name="connected_to_server_message">Connected to the server Successfully</string>
```
Comment (Rimjhim28) [inline openmrs-client/src/main/res/values/strings.xml]: Same string resource added twice. One of them can be removed

Comment (Abdelaty) [reply openmrs-client/src/main/res/values/strings.xml]: I've removed it and I will make a new pull request to avoid many unnecessary commits.

Comment (VANKINEENITAWRUN) [reply openmrs-client/src/main/res/values/strings.xml]: @Abdelaty Follow this [Link](https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit)  to squash Commits. Also, run the below command in the base directory 
`./gradlew test` 
before making PR, to ensure Travis build works.

**Comment by VANKINEENITAWRUN** [comment]
@Abdelaty once you have made your's commit try running
 `git pull --rebase upstream master` solve the conflicts. Then Squash the commits to one as per the PR [Guidelines](https://wiki.openmrs.org/display/docs/Pull+Request+Tips) then make a PR

File: openmrs-client/src/main/res/values/strings.xml (hunk 1 above)
Comment (Abdelaty) [reply openmrs-client/src/main/res/values/strings.xml]: @VANKINEENITAWRUN  thank you for your reply. I've used it but got some issues in tests, so I made a new pull request with one commit. 
Sorry for the conflict guys @VANKINEENITAWRUN @Rimjhim28 @f4ww4z 
I will try to fade this in the feature.

**Pull request closed without merging**